tests/terraform/graph/utils/test_utils.py

```diff
@@ -183,21 +180,3 @@ def test__attribute_has_nested_attributes_list(self):
         self.assertTrue(attribute_has_nested_attributes(attribute_key='filter.1.values', attributes=attributes))
         self.assertFalse(attribute_has_nested_attributes(attribute_key='filter.1.values.0', attributes=attributes))
 
-    def test_malicious_eval(self):
```

Comment:
@rotemavni Did we move this somewhere?

Reply:
@nimrodkor no, we removed the `decode_graph_property_value` so this logic doesn't need to be tested.
We still have the logic of avoid evaluating strings with double underscores, which protects us even better.